app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -223,6 +223,11 @@ def parse_pod(pod)
         end
       end
 
+      conditions = pod.status.Condition
```

Comment:
In #3643 we're moving to v1 (/cc @oschreib) so this will become `pod.status.conditions`. We'll have to sync on the order of the patches.